File: Python/tests/test_dd_cs_single.py
import pytest

from hbayesdm.models import dd_cs_single


def test_dd_cs_single():
    _ = dd_cs_single(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_dd_cs_single.R
context("Test dd_cs_single")
library(hBayesDM)

test_that("Test dd_cs_single", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(dd_cs_single(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
